plone/namedfile/scaling.py

```diff
@@ -192,18 +193,18 @@ def create_scale(self, data, direction, height, width, **parameters):
         )
 
     def __call__(
-        self,
-        fieldname=None,
-        direction='thumbnail',
-        height=None,
-        width=None,
-        scale=None,
-        **parameters
+            self,
+            fieldname=None,
+            direction='thumbnail',
+            height=None,
+            width=None,
+            scale=None,
+            **parameters
     ):
 
         """Factory for image scales`.
         """
-        orig_value = getattr(self.context, fieldname)
+        orig_value = getattr(self.context, fieldname, None)
```

Comment:
this is redundant, or only explicit. getattr will always return None if the attribute could not be found.